src/app/(shop)/womens/[id]/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function page({ params }: { params: { slug: string } }) {
```

Comment:
Should the param here be a number instead of a string?

<img width="240" alt="image" src="https://github.com/Jwensloff/JSJStorefront/assets/130028791/9edd5d28-012d-42d2-b242-2e9a6ec49e13">

Reply:
Probably? I'm honestly not super confidence on how to handle these yet, but I changed them to number since we are planning on passing the ids!